homework/lesson_number_2/lesson_number_2/Program.cs

```diff
@@ -0,0 +1,214 @@
+            PrintUpdatedArray(a);
+            
+            //send in correct username/password
+            string loginMsg = Login("Username", "Passw0rd");
+            Console.WriteLine(loginMsg);
+        }
+
+        private static void PrintUpdatedArray(int[] g)
+        {
+            for (int i = 0; i < g.Length; i++)
+            {
+                Console.WriteLine(g[i]);
+            }
+        }
+
+        private static string Login(string savedUsername, string savedPassword)
+        {
+            string loginMsg = "";
+            
+            Console.WriteLine("Please enter username: ");
+            string username = Console.ReadLine();
+
+            Console.WriteLine("Please enter password: ");
+            string password = Console.ReadLine();
+
+            if (username.Equals(savedUsername, StringComparison.OrdinalIgnoreCase) &&
+                password.Equals(savedPassword, StringComparison.Ordinal))
+            {
+                loginMsg = "Logged In!";
+                return loginMsg;
+            }
+
+            loginMsg = "Username or password was incorrect!";
+            return loginMsg;
+        }
+
+        static int[] SortArray(int[] numbers)
+        {
+            int[] newArray = new int[numbers.Length];
+            
+            for (int i = 0; i < numbers.Length; i++)
+            {
+                var num = FindNewLowestNumber(numbers, newArray);
+
+                newArray[i] = num;
+            }
+            return newArray;
+        }
+
+        private static int FindNewLowestNumber(int[] numbers, int[] newArray)
+        {
+            //So first comparison is always set to num
+            int num = 999;
+
+            for (int e = 0; e < numbers.Length; e++)
+            {
+                if (numbers[e] < num)
+                {
+                    int lowestNumber = numbers[e];
+                    bool numberAlreadyInNewArray = CheckIfNumberAlreadyInNewArray(lowestNumber, newArray, e);
+
+                    if (!numberAlreadyInNewArray) num = lowestNumber;
+                }
+            }
+
+            return num;
+        }
+
+        private static bool CheckIfNumberAlreadyInNewArray(int lowestNumber, int[] newArray, int e)
+        {
+            bool numberAlreadyInNewArray = false;
+
+            foreach (var newArrayNum in newArray)
+            {
+                if (lowestNumber == newArrayNum)
+                {
+                    numberAlreadyInNewArray = true;
+                }
+            }
+
+            return numberAlreadyInNewArray;
+        }
+
+        static int[] AddElementEndOfArray(int[] numbers, int newNum)
+        {
+            int[] newArray = new int[numbers.Length + 1];
+
+            for (int i = 0; i < newArray.Length; i++)
+            {
+                if (i + 1 == newArray.Length)
+                {
+                    newArray[i] = newNum;
+                    continue;
```

Comment:
A `continue` is ok here (and I think this is how Kai taught it) but I'd presonally use an `else` statement if there is only one other line after the continue (e.g. if it is a simple if x do y else do z)